src/styled/styledBalanceForm.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {Component} from 'react'
+import styled, {keyframes} from 'styled-components';
+
+
+const Container = styled.div`
```

Comment:
Could you specify it a little better?
From what I see it is centering and middle -ing.
Wouldn't it be then named MiddleCenteringContainer?